File: wls-eai-service/src/main/java/de/muenchen/oss/wahllokalsystem/eaiservice/rest/wahlvorschlag/model/Wahlvorschlaege.java
package de.muenchen.oss.wahllokalsystem.eaiservice.rest.wahlvorschlag.model;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import java.util.Set;

public record Wahlvorschlaege(@NotNull String wahlbezirkID,
                              @NotNull String wahlID,
                              @NotNull String stimmzettelgebietID,
                              @NotNull @Size(min = 1) Set<Wahlvorschlag> wahlvorschlaege) {
}

File: wls-gui-wahllokalsystem/src/types/wahlvorschlaege/Wahlvorschlaege.ts
import type { Wahlvorschlag } from "@/types/wahlvorschlaege/Wahlvorschlag.ts";

export interface Wahlvorschlaege {
  wahlID: string;
  wahlbezirkID: string;
  stimmzettelgebietID: string;
  wahlvorschlaege: Wahlvorschlag[];
}
